packages/cli/package.json

```diff
@@ -97,6 +97,7 @@
       "rimraf": "^3.0.1",
       "source-map-support": "^0.5.16",
       "typescript": "~3.7.5",
+      "tslib": "^1.10.0",
```

Comment:
No need to add here as `cli` is written in plain js.

Reply:
@raymondfeng I've removed `tslib` from `dependency`. AFAIK the one you've quoted `config.templateDependencies` is for the generating the projects.

Reply:
Oh, yes. I was confused.